lib/active_model_serializers.rb

```diff
@@ -2,6 +2,7 @@
 require 'active_support'
 require 'active_support/core_ext/object/with_options'
 require 'active_support/core_ext/string/inflections'
+require 'active_support/json'
```

Comment:
is https://github.com/rails/rails/blob/v5.0.0.beta3/activesupport/lib/active_support/json.rb

```ruby
require 'active_support/json/decoding'
require 'active_support/json/encoding'
```